lib/bolt/inventory/options.rb

```diff
@@ -0,0 +1,136 @@
+# frozen_string_literal: true
+
+require 'bolt/config/options'
+
+module Bolt
+  class Inventory
+    module Options
+      # Top-level options available in the inventory.
+      OPTIONS = %w[
+        config
+        facts
+        features
+        groups
+        targets
+        vars
+        version
+      ].freeze
+
+      # Definitions used to validate the data.
+      # https://github.com/puppetlabs/bolt/blob/main/schemas/README.md
+      DEFINITIONS = {
+        "alias" => {
+          description: "A unique alias to refer to the target. Aliases cannot conflict "\
+                       "with the name of a group, the name of a target, or another alias.",
+          type: Array,
+          uniqueItems: true,
+          items: {
+            type: String,
+            _plugin: true
+          },
+          _plugin: true
+        },
+        "config" => {
+          description: "A map of configuration options.",
+          type: Hash,
+          properties: {},
+          _plugin: true
+        },
+        "facts" => {
+          description: "A map of system information, also known as facts, for the target.",
+          type: Hash,
+          _plugin: true
+        },
+        "features" => {
+          description: "A list of available features for the target.",
+          type: Array,
+          uniqueItems: true,
+          items: {
+            type: String,
+            _plugin: true
+          },
+          _plugin: true
+        },
+        "groups" => {
+          description: "A list of groups and their associated configuration.",
+          type: Array,
+          items: {
+            type: Hash,
+            required: ["name"],
+            properties: {
+              "config"   => { _ref: "config" },
+              "facts"    => { _ref: "facts" },
+              "features" => { _ref: "features" },
+              "groups"   => { _ref: "groups" },
+              "name"     => { _ref: "name" },
+              "targets"  => { _ref: "targets" },
+              "vars"     => { _ref: "vars" }
+            },
+            _plugin: true
+          },
+          _plugin: true
+        },
+        "name" => {
+          description: "A human-readable name to refer to the group or target. Names "\
+                       "cannot conflict with the name of a group, the name of a target, "\
+                       "or the alias of a target. A name is required for a group and is "\
+                       "required for a target unless the uri option is set.",
+          type: String,
+          _plugin: true
+        },
+        "plugin_hooks" => {
```

Comment:
Ugh. It'd be nice to rename this, but that seems like an inventory v3 thing?

Reply:
We can do that as part of 3.0, we obviously need to do it soon.